src/collections/blog/2025/01-01-meet-the-maintainer-sudhanshu-dasgupta/post.mdx

```diff
@@ -0,0 +1,229 @@
+     Also all credits to the team like how they supported and provided feedback at every step that helped me a lot to gain this achivement.
+     So Meshery is a very interesting project and how it solves the issue of cloud native infrastructure. It just amazes me and just makes me feel that there is a lot to learn and indulge in this area and one of interesting fields so this all made me stay.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    You’re a <Link to="/cloud-native-management/meshery">Meshery</Link> maintainer and have been for some long time now. What does being a Meshery maintainer mean to you?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
+  <p>
+    Yes this is a great question. As a maintainer the first most important thing is the responsibility and success of the project. You are responsible for quality work that should be applied to the project. As a maintainer I try to provide quality reviews to the contributors and seek that they get their PRs merged. Being a maintainer is about building not just a tool or feature but taking ownership of the project's vision, leading development efforts, and mentoring new contributors to help them find their footing in open source.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    Have you worked with any other open source project? How does Layer5 compare?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
+  <p>
+     I have worked with a few other projects but they were not so large scale projects.
+     In comparison with Layer5, it is the community. Open source projects don't only mean how big the project is or how much you code. It is how engaged you are and about collaboration. This is the speciality of Layer5.
+     You will see meshmate and community managers helping though your way and getting success.  In other projects I don't know if you will get such one to one interactions often.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    <Link to="/projects">Layer5 projects</Link> have a number of active, open
+    source projects. You’ve been consistently contributing to a few of them.
+    Which one(s) are you currently focusing on?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
+  <p>
+    Ok so there are many I am focusing on. One of them is of course Meshery itself. We are building the feature of multi-tenancy of environments and workspaces. The second is Sistent which is Layer5's own design system to migrate all components to this sistent.
+    And the one is Layer5's own site which is running on gatsby so helping in getting prs reviewed there and merged and some dependency migration is also going on which i am focusing on.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>What’s the coolest Meshery demo you have done/seen?</p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
```

Comment:
@vishalvivekm, I thought that we had enhanced the Meet the Maintainers blog series to templatize interviewer and interviewee names, such they didn't have to be repeated other than in front matter. If we haven't, that's OK. If we have, we can use it here.